pyincore/analyses/meandamage/meandamage.py

```diff
@@ -30,17 +30,17 @@ def get_spec(self):
                     'description': 'result dataset name',
                     'type': str
                 },
+                {
+                    'id': 'damage_interval_keys',
+                    'required': True,
+                    'description': 'Column name of the damage interval must be four and ranged in order',
+                    'type': (list, str)
```

Comment:
hmm this representation might be a little bit hard to understand. Could we do `list<str>` instead? You are parsing the tuple anyway